appveyor.yml

```diff
@@ -23,10 +23,12 @@ install:
 build_script:
   - cmd: cd python
   - cmd: python setup.py build_ext --inplace
+  # Install some modules needed for tests
   - cmd: python -m pip install PyVCF
   - cmd: python -m pip install newick
   - cmd: python -m pip install python_jsonschema_objects
   - cmd: python -m pip install xmlunittest
+  - cmd: python -m pip install msgpack
```

Comment:
msgpack is on conda-forge, so we can put this requirement into ``conda-minimal.txt``. https://anaconda.org/conda-forge/msgpack-python

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/543/commits/1d8d05c6b7d6690d90e056f5b9fa7ed335c53339